packages/babel-plugin-proposal-do-expressions/src/index.ts

```diff
@@ -1,4 +1,6 @@
 import { declare } from "@babel/helper-plugin-utils";
+import traverse, { type NodePath } from "@babel/traverse";
+import * as t from "@babel/types";
```

Comment:
To reduce dependencies duplication, we generally access traverse and types from the `api` object injected by `@babel/core`: `const { types: t, traverse } = api`.